requirements.txt

```diff
@@ -24,6 +25,7 @@ django-anymail[mailgun]==1.2
 django-cors-headers==2.1.0
 django-redis==4.7.0
 django-server-status==0.5.0
+django-storages-redux==1.3.2
```

Comment:
I think this should just be `django-storages` now since this package is no longer getting updates in favor of the official one

Reply:
The latest version of `django-storages` no longer includes `storages.backends.overwrite.OverwriteStorage` specified in `settings.py` if S3 isn't enabled.  Everything seems to still work fine if I remove this `else` block, and tests pass.  

```python
else:
    # by default use django.core.files.storage.FileSystemStorage with
    # overwrite feature
    DEFAULT_FILE_STORAGE = 'storages.backends.overwrite.OverwriteStorage'
```